src/configure/helper/LocalGitRepoHelper.ts

```diff
@@ -197,4 +197,10 @@ export class LocalGitRepoHelper {
             manifestFilePath);
         await vscode.window.showTextDocument(vscode.Uri.file(manifestFilePath));
     }
+
+    public async readFileConetent(pathToFile: string): Promise<string>{
+        await vscode.workspace.saveAll(true);
```

Comment:
why doing the extra save here? If user has saved, we should take that.

Reply:
There can be scenario, where user just clicked on Modify and Push button without saving the file. So we are saving it and then sending to provisioning service.

Reply:
@tiwarishub when we display draft files to user, we would have already saved the files. is that right? if so, then we will let user save the changes he made, otherwise just commit the files which we saved earlier.